src/Toml/Bi/Map.hs

```diff
@@ -417,24 +433,28 @@ instances.  Usually used as 'Toml.Bi.Combinators.enumBounded' combinator.
 -}
 _EnumBounded :: (Show a, Enum a, Bounded a) => TomlBiMap a AnyValue
 _EnumBounded = _EnumBoundedText >>> _Text
+{-# INLINE _EnumBounded #-}
 
 {- | 'Word' bimap for 'AnyValue'. Usually used as
 'Toml.Bi.Combinators.word' combinator.
 -}
 _Word :: TomlBiMap Word AnyValue
 _Word = _BoundedInteger >>> _Integer
+{-# INLINE _Word #-}
 
 {- | 'Int' bimap for 'AnyValue'. Usually used as
 'Toml.Bi.Combinators.int' combinator.
 -}
 _Int :: TomlBiMap Int AnyValue
 _Int = _BoundedInteger >>> _Integer
+{-# INLINE _Int #-}
 
 {- | 'Float' bimap for 'AnyValue'. Usually used as
 'Toml.Bi.Combinators.float' combinator.
 -}
 _Float :: TomlBiMap Float AnyValue
 _Float = iso realToFrac realToFrac >>> _Double
+{-# INLINE _Float #-}
 
 -- | Helper bimap for 'Data.Text.Text' and strict 'ByteString'
 _ByteStringText :: TomlBiMap ByteString Text
```

Comment:
I think it's actually a good idea to inline `_ByteStringText` as well because optimizations on ByteString and Text functions are relying on inlining and can be huge in some cases.